scripts/make-live.R

```diff
@@ -61,12 +61,18 @@ infiles <- c(file.path(root_dir, "intro-to-R-tidyverse",
 
 # Rerender notebooks if --rendering is TRUE
 if (opt$rendering) {
-  purrr::map(infiles, rmarkdown::render, envir = new.env(), quiet = TRUE)
+  # capture to avoid printing to stdout
+  rendered <-purrr::map(infiles, 
+                        function(f){
+                          # mini function to track progress
+                          message("Rendering ", stringr::str_replace(f, paste0(root_dir, "/"), "")) # hide root
+                          rmarkdown::render(f, envir = new.env(), quiet = TRUE)
+                        })
 }
 
 # new files will be made with -live.Rmd suffix
 outfiles <- stringr::str_replace(infiles, "(.*)\\.Rmd$", "\\1-live.Rmd")
 
 # Generate live versions
-# capture to avoid printing to stdout
+
```

Comment:
```suggestion
# capture to avoid printing to stdout
```

This change seemed inadvertent to me, but not a big deal either way.

Reply:
It was inadvertent, I must have cut when I meant to copy.